test/util.ts

```diff
@@ -1,6 +1,7 @@
 /* eslint-disable jest/no-standalone-expect */
 import * as nativeAssert from "assert";
-import { lauxlib, lua, lualib, to_jsstring, to_luastring } from "fengari";
+import { lauxlib, lua, lualib } from "lua-wasm-bindings/dist/lua.54";
```

Comment:
As we discussed on discord, please use 5.3 for tests as long as we do not officially support 5.4 yet. (I'd also be fine with another PR that adds 5.4 as supported LuaTarget)